### ISSUE TITLE
Trader-Bugmark User Issue 3

### ISSUE THREAD

**Issue by NDamen** [bugmark/test_bench#106]
![image](https://user-images.githubusercontent.com/25160740/41741681-64a61010-7561-11e8-9c95-d5b89c6783c4.png)

Issue Source: http://github.com/bugmark-trial/trader1/issues/313

Confusion: Payout resolution to a different person than Funder OR Worker.

My first guess was that it’s because Don resold their part and since the issue didn’t get resolved they are seen as the last funder, thus they get the payout? However; a) The issue did get resolved, therefore the worker should have received the payout (Pull123).

b) This situation creates multiple funders. If it gets resolved this way, the worker turned funder will always win; if it does resolve they get paid, if it doesn’t resolve they get paid as well. Regardless, information should be provided on which funder gets pay-out when the issue is closed; the original funder as listed under funder, or the last funder as listed under worker (person who is reselling their work).

**Comment by andyl** [comment]
Yeah - same as #105.  Introducing resell offers adds another layer of complexity to the explanation process.  The reselling mechanics is represented as a graph of position ownership, auditable and blockchain compatible.  We;ve been better at the algorithms than the human explaination - thanks for the feedback we'll keep working on this.
